basin3d/plugins/usgs.py

```diff
@@ -280,6 +289,29 @@ def _load_point_obj(datasource, json_obj, feature_observed_properties, synthesis
         return monitoring_feature
 
 
+def parse_sites_response(usgs_site_response):
```

Comment:
Make this function private if it is not used outside of this module.  Additionally, it is good practice to make any function intended for internal module use as private.
```suggestion
def _parse_sites_response(usgs_site_response):
```